File: LeetCode Solutions/Problems/1078. Occurrences After Bigram.py
class Solution:
    def findOcurrences(self, text: str, first: str, second: str) -> List[str]:
        arr = []
        token = text.split(" ")
        for i in range(2, len(token)):
            if token[i-2] == first and token[i-1] == second:
                arr.append(token[i])
        return arr

File: LeetCode Solutions/Problems/1078. Occurrences After Bigram.java
class Solution {
    public String[] findOcurrences(String text, String first, String second) {
        String[] s = text.split(" ");
        List<String> arr = new ArrayList<String>();
        for (int i=2; i<s.length; ++i){
            if (s[i-2].equals(first) && s[i-1].equals(second)){
                arr.add(s[i]);
            }
        }
        return arr.toArray(new String[0]);
    }
}